compiler/src/latex_printer.ml

```diff
@@ -22,9 +22,19 @@ let dname = latex "dname"
 let pannot = latex "annotation"
 let pprim = latex "primitive"
 let arrow = symbol "arrow"
+
 let sharp fmt () = F.fprintf fmt "\\#"
 let openbrace fmt () = F.fprintf fmt "\\{"
 let closebrace fmt () = F.fprintf fmt "\\}"
+let ampersand fmt () = F.fprintf fmt "\\&"
```

Comment:
This should not be needed.

Reply:
My intention was that in the future we might have annotations that take strings and we want to print them. Should I restrict this to cover only the characters we allow inside annotations we support? (but I'm not sure how to check exhaustively all the annotations we support...)

Reply:
Escaping ampersand is not needed, look at the printing of bitwise and operators below.

Reply:
done